pom.xml

```diff
@@ -72,18 +85,4 @@
 			</plugin>
 		</plugins>
 	</build>
-
-
-	 <distributionManagement>
```

Comment:
@joaofreitas-qubit notice that this project is not a child of our open source parent pom, this means it does not contain any distribution management by default, removing this will fail any future releases.

Reply:
you are right we forgot to delete this PR